amazon-msk/cdktf/secure-public-access/main.ts

```diff
@@ -132,22 +131,7 @@ export class ZillaPlusSecurePublicAccessStack extends TerraformStack {
       });
     }
 
-    const awsccMskCluster = new DataAwsccMskCluster(this, "awsccMskCluster", {
-      id: mskCluster.id
-    })
-
-    const mtlsEnabled = Fn.lengthOf(awsccMskCluster.clientAuthentication.tls.certificateAuthorityArnList) > 0;
-
     let mskClientAuthentication = userVariables.mskClientAuthentication;
-    if (userVariables.mskClientAuthentication === "Unknown") {
-      mskClientAuthentication = mtlsEnabled
-        ? "mTLS"
-        : mskCluster.bootstrapBrokersSaslScram
-        ? "SASL/SCRAM"
-        : mskCluster.bootstrapBrokers
-        ? "Unauthorized"
-        : userVariables.mskClientAuthentication;
-    }
```

Comment:
We are no longer detecting the compatible authentication strategy for Zilla to connect to MSK cluster?